java/sidecar/src/main/java/com/instaclustr/jersey/JerseyServerModule.java

```diff
@@ -28,17 +51,94 @@ protected void configure() {
     }
 
     @Provides
-    HttpServer provideHttpServer(final ResourceConfig resourceConfig) throws UnknownHostException, URISyntaxException {
-        // TODO: make URI configurable
-        String protocol = "http://";
-        String port = ":4567/";
-        String hostname = InetAddress.getLocalHost().getHostName();
-        if (hostname.equals("127.0.0.1"))
-        {
-            hostname = "localhost";
+    HttpServer provideHttpServer(final ResourceConfig resourceConfig) {
+
+        logger.info("Listening on " + httpServerUri);
+
+        return JdkHttpServerFactory.createHttpServer(httpServerUri, resourceConfig, false);
+    }
+
+    @Singleton
+    @Provides
+    ObjectMapper provideObjectMapper() {
```

Comment:
We should move this to a separate module, since Jackson configuration has nothing to do with the Jersey server.